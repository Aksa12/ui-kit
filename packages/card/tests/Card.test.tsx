import React from "react";

import { Card } from "../";
import { render } from "enzyme";
import toJSON from "enzyme-to-json";

describe("Card", () => {
  it("default", () => {
    expect(toJSON(render(<Card>Example Content</Card>))).toMatchSnapshot();
  });
});
