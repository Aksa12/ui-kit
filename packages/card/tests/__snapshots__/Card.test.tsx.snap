// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card default 1`] = `
<div
  class="css-1rmvfmi"
>
  <div
    class="css-2yjhih"
  >
    Example Content
  </div>
</div>
`;

exports[`Card with aspectRatio set 1`] = `
<div
  class="css-88gmn3"
>
  <div
    class="css-2yjhih"
  >
    Example Content
  </div>
</div>
`;

exports[`Card with paddingSize set 1`] = `
<div
  class="css-1rmvfmi"
>
  <div
    class="css-jl90x"
  >
    Example Content
  </div>
</div>
`;
